omc3/hole_in_one.py

```diff
@@ -35,15 +35,16 @@
 from datetime import datetime
 from os.path import abspath, basename, dirname, join
 
+import turn_by_turn as tbt
```

Comment:
Is this addition needed?

Reply:
no idea how it ended up there

Reply:
but yes, it is needed below... wtf?

Reply:
oh, it moved. I don't know why it moved. some order-the-imports magic